src/QrCode.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+
+namespace Razorpay.Api
+{
+    public class QrCode : Entity
+    {
+        new public QrCode Fetch(string id)
+        {
```

Comment:
`string.Format("{0}/payments/qr_codes/ `
This is repeating in every function. Please move it to constructor.

Reply:
i created `QrCodeUrl` method for this which return url . because when i try to created construtor i am not able to get url in while using chain method.